js/src/ui/Form/AddressSelect/addressSelectStore.js

```diff
@@ -194,6 +239,8 @@ export default class AddressSelectStore {
           .filter((result) => result && !ZERO.test(result.address));
```

Comment:
Might actually want to get rid of `this.registryValues = [];` line 231, to prevent the flickering when typing an autocompleted name.